documentcloud/documents/tests/test_oembed.py

```diff
@@ -0,0 +1,335 @@
+        self.assertIn('<iframe src="', response["html"])
+        self.assertIn(
+            f"{settings.DOCCLOUD_EMBED_URL}/documents/123-test-document/?responsive=1",
+            response["html"],
+        )
+        self.assertIn(
+            'title="Test Document (Hosted by DocumentCloud)"', response["html"]
+        )
+        self.assertIn('width="600" height="400"', response["html"])
+        self.assertIn(
+            'style="border: 1px solid #aaa; width: 100%; height: 800px;',
+            response["html"],
+        )
+        self.assertIn(
+            'sandbox="allow-scripts allow-same-origin allow-popups allow-forms',
+            response["html"],
+        )
+
+    def test_document_oembed_get_dimensions(self):
+        """Test the get_dimensions method of DocumentOEmbed"""
+        # Test with both max_width and max_height - should preserve both dimensions
+        width, height = self.document_oembed.get_dimensions(self.document, 600, 500)
+        self.assertEqual(width, 600)
+        self.assertEqual(height, 500)
+
+        # Test with max_width less than default
+        width, height = self.document_oembed.get_dimensions(self.document, 500, None)
+        self.assertEqual(width, 500)
+        self.assertEqual(height, 333)  # 500/1.5 = 333.33...
+
+        # Test with only max_height
+        width, height = self.document_oembed.get_dimensions(self.document, None, 450)
+        self.assertEqual(width, 675)  # 450*1.5 = 675
+        self.assertEqual(height, 450)
+
+        # Test with no max dimensions
+        width, height = self.document_oembed.get_dimensions(self.document, None, None)
+        self.assertEqual(width, 800)  # Should be default
+        self.assertEqual(height, 533)  # 800/1.5 = 533.33...
+
+    def test_document_oembed_get_context(self):
+        """Test the get_context method of DocumentOEmbed"""
+        query = Query("param=value")
+        extra = {"width": 600, "height": 400, "style": ""}
+
+        context = self.document_oembed.get_context(self.document, query, extra)
+
+        expected_src = (
+            f"{settings.DOCCLOUD_EMBED_URL}/documents/123-test-document/?param=value"
+        )
+        self.assertEqual(context["src"], expected_src)
+        self.assertEqual(context["width"], 600)
+        self.assertEqual(context["height"], 400)
+        self.assertEqual(context["style"], "")
+
+    def test_document_oembed_get_style(self):
+        """Test the get_style method of DocumentOEmbed"""
+        # Test responsive with no max dimensions
+        style = self.document_oembed.get_style(None, None)
+        self.assertEqual(
+            style, " width: 100%; height: 800px; height: calc(100vh - 100px);"
+        )
+
+        # Test with max_width only
+        style = self.document_oembed.get_style(600, None)
+        self.assertEqual(
+            style,
+            " width: 100%; height: 800px; height: calc(100vh - 100px); max-width: 600px;",  # pylint: disable=line-too-long
```

Comment:
Can you just break this string up onto two lines instead of disabling the line too long check?